### DISCUSSION TITLE
posts/2022/07/more-tips-using-giscus/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#19] [Announcements]
# posts/2022/07/more-tips-using-giscus/

How to lessen the load or, at least, make it easier to bear.

https://www.brycewray.com/posts/2022/07/more-tips-using-giscus/

**Comment by PersonMeetup** [comment]
From what I experienced lazy loading appears to be broken in Firefox, with Giscus making GET calls before I clicked the `<details>` button. Everything works fine in Chrome though, which makes me wonder if it's something on my end.

Either way, I found this article incredibly helpful for mentioning the `<details>` and `<summary>` elements, as they solve a problem I was about to heavily over-complicate.

**Reply by brycewray** [reply]
@PersonMeetup Yes, lazy-loading on Firefox also is problematic, as I noted is the case for `preconnect` and `dns-prefetch`. I probably should mention that in one of these posts. Thanks for the kind words!

**Note**: Have now edited the [original post](https://www.brycewray.com/posts/2022/05/tips-using-giscus/#get-lazy) to note this — and it turns out it’s true for Safari, too. :(
